specs/WebMessageObjects.md

```diff
@@ -171,18 +402,82 @@ namespace Microsoft.Web.WebView2.Core
         /// Any DOM object type that can be natively representable that has been passed in to
         /// `additionalObjects` parameter will be accessible here.
         /// Currently a WebMessage object can be the following type:
-        /// $net$
-        /// - System.IO.FileInfo
-        /// $end$
-        /// $winrt$
-        /// - Windows.Storage.StorageFile

```

Comment:
Wait, are we removing support for these types and replacing them with the two new types?